lib/views/home/widgets/announcement/announcement_widget.dart

```diff
@@ -103,16 +76,13 @@ class _AnnouncementWidgetState extends ConsumerState<AnnouncementWidget> {
             if (widget.onPressedDismiss != null) {
               widget.onPressedDismiss!();
             }
-            _toggleCollapse();
             _handleTrackEvent(
               ref,
               announcement.id,
               StringConstants.dismiss.toLowerCase(),
             );
-            ref.invalidate(remoteStatsProvider);
-            ref.read(remoteStatsProvider);
-            ref.invalidate(homeProvider);
-            ref.read(homeProvider);
+            //ignore: unused_result
+            // ref.refresh(homeProvider);
```

Comment:
@osamaasifoak I'm going to merge all these PRs if they don't have conflicts but please make a note to fix all these things, like commented out code and submit a new PR with the fixes 👍